exception/Exception.js

```diff
@@ -83,6 +83,14 @@ export class Exception {
         return this.createInternal(defaults, {});
     }
 
+    static serverUnavailable(requestOptions) {
+        return this.createInternal({
+            name: 'Server Unavailable',
+            message: 'The application server is not available.',
```

Comment:
Do we want to say more?  like:  "If your internet connection is good and this condition persists, please contact ...